Clojure/Default.sublime-keymap

```diff
@@ -0,0 +1,26 @@
+[
```

Comment:
This file should use tabs for indentation

Reply:
Fixed.